sophiau added

berrakozer opened the pull request:
work in progress

berrakozer commented:
@sbillinge the list is completely added. Pull Request ready to go.

sbillinge commented on db/institutions.yml:
```diff
@@ -834,6 +856,36 @@ mpi:
+  country: germany
+  day: 17
+  month: 3
+  name: mpihamburg
```
Max Planck Institute for the Structure and Dynamics of Matter

sbillinge commented on db/institutions.yml:
```diff
@@ -834,6 +856,36 @@ mpi:
+  uuid: 4bc411c5-6e58-4b2e-b765-7d72403ee42d
+  year: 2020
+mpihamburg:
+  aka: []
```
MPI-SDM
MPISDM
MPI SDM
mpihamburg

sbillinge commented on db/institutions.yml:
```diff
@@ -834,6 +856,36 @@ mpi:
+  country: germany
+  day: 17
+  month: 3
+  name: mpissr
```
Max Planck Institute for Solid State Research

sbillinge commented on db/institutions.yml:
```diff
@@ -834,6 +856,36 @@ mpi:
+  uuid: 7a4d8908-41f3-4681-9690-19567a49b22a
+  year: 2020
+mpissr:
+  aka: []
```
MPI-SSR
MPISSR
MPI SSR

sbillinge commented on db/institutions.yml:
```diff
@@ -872,6 +924,40 @@ natautoumexico:
+  zip: '20375'
+navigant:
+  aka: []
+  city: new york city
```
New York

sbillinge commented on db/institutions.yml:
```diff
@@ -1107,6 +1316,18 @@ rutherfordappletonlaboratory:
   uuid: d48cb08d-d9a9-4f86-b12d-5751df57006a
   year: 2020
+sandianationallaboratories:
+  aka: []
```
SNL

sbillinge commented on db/institutions.yml:
```diff
@@ -1142,6 +1396,19 @@ srtmu:
+  country: usa
+  day: 17
+  month: 3
+  name: stony brook university
```
State University of New York, Stony Brook

sbillinge commented on db/institutions.yml:
```diff
@@ -1142,6 +1396,19 @@ srtmu:
   zip: ''
+stonybrooku:
+  aka:
+    - sbu
```
add 
suny-sb
sunysb

sbillinge commented:
@berrakozer I am afraid this will need a rebase also.  Please get help with that, don't try it on your own.  Find someone in the group and do a zoom with shared screen or sthg like that.  I can do the Zoom if you don't find anyone.

berrakozer commented:
Hello Professor,

I’m sorry for the late response was feeling slightly tired literally took
vitamins and forced myself to take a full sleep, ended up waking up
late...I’m sorry. Can I make the edits from pycharm (just is it possible to
one by one fix all of them manually so fast?) if yes am i committing inside
pycharm and on terminal push origin? If not can you tell me how can
modifications be executed for each institution without committing changes
each time from terminal? And how much time do we have before you send it?

If its easier can i just call you from zoom and sort this out?

Best,

Berrak

On Tue, Mar 17, 2020 at 06:09 Simon Billinge <notifications@github.com>
wrote:

> *@sbillinge* commented on this pull request.
>
> Thanks so much @berrakozer <https://github.com/berrakozer> !
>
> I am afraid that in here we need the proper full name with correct
> capitalization though, so Geneva and Switzerland and McKinsey & Company
> etc etc. The best way to fix this is probably to edit the file directly.
> You don't need to make a commit for each entry either, so you could save
> time by making fewer commits. Please keep the id lower case though. By way
> of examplemit:
>
>
>   aka:
>
>     - MIT
>
>   city: Boston
>
>   country: USA
>
>   day: 17
>
>   month: 3
>
>   name: Massachusetts Institute of Technology
>
>   state: MA
>
>   updated: 2020-03-17 02:30:29.709566
>
>   uuid: 8acc5248-f9f0-4b26-8deb-3f7dd54c041e
>
>   year: 2020
>
>   zip: '02139'
>
>
> I am afraid pretty much all of the entries need to be modified in this
> sense. I will also add some in-line comments if I see any other issues.
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -1023,6 +1142,52 @@ pekingu:
>
>    state: ''
>
>    street: 5 Yiheyuan Road
>
>    zip:  <https://www.google.com/maps/search/5+Yiheyuan+Road%0D%0D%0A+++zip:++100871?entry=gmail&source=g>'100871 <https://www.google.com/maps/search/5+Yiheyuan+Road%0D%0D%0A+++zip:++100871?entry=gmail&source=g>'
>
> +pennstateu:
>
> +  aka: []
>
> +  city: Pennsylvania
>
>
> City College
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -834,6 +856,36 @@ mpi:
>
>        name: Institute for Solid State Physics
>
>    name: Max Planck Institute
>
>    state: ''
>
> +mpicps:
>
> +  aka: []
>
> +  city: dresden
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpicps
>
>
> Max Planck Institute for Chemical Physics of Solids then put MPI-CPS,
> MPICPS and MPI CPS as aka's
> ------------------------------
>
> In db/institutions.yml:
>
> > +  aka: []
>
> +  city: dresden
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpicps
>
> +  updated: 2020-03-17 02:27:00.262253
>
> +  uuid: 4bc411c5-6e58-4b2e-b765-7d72403ee42d
>
> +  year: 2020
>
> +mpihamburg:
>
> +  aka: []
>
> +  city: hamburg
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpihamburg
>
>
> Max Planck Institute for the Structure and Dynamics of Matter
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -834,6 +856,36 @@ mpi:
>
>        name: Institute for Solid State Physics
>
>    name: Max Planck Institute
>
>    state: ''
>
> +mpicps:
>
> +  aka: []
>
> +  city: dresden
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpicps
>
> +  updated: 2020-03-17 02:27:00.262253
>
> +  uuid: 4bc411c5-6e58-4b2e-b765-7d72403ee42d
>
> +  year: 2020
>
> +mpihamburg:
>
> +  aka: []
>
>
> MPI-SDM
> MPISDM
> MPI SDM
> mpihamburg
> ------------------------------
>
> In db/institutions.yml:
>
> > +  aka: []
>
> +  city: hamburg
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpihamburg
>
> +  updated: 2020-03-17 02:23:15.063435
>
> +  uuid: 7a4d8908-41f3-4681-9690-19567a49b22a
>
> +  year: 2020
>
> +mpissr:
>
> +  aka: []
>
> +  city: stuttgart
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpissr
>
>
> Max Planck Institute for Solid State Research
> ------------------------------
>
> In db/institutions.yml:
>
> > +  name: mpicps
>
> +  updated: 2020-03-17 02:27:00.262253
>
> +  uuid: 4bc411c5-6e58-4b2e-b765-7d72403ee42d
>
> +  year: 2020
>
> +mpihamburg:
>
> +  aka: []
>
> +  city: hamburg
>
> +  country: germany
>
> +  day: 17
>
> +  month: 3
>
> +  name: mpihamburg
>
> +  updated: 2020-03-17 02:23:15.063435
>
> +  uuid: 7a4d8908-41f3-4681-9690-19567a49b22a
>
> +  year: 2020
>
> +mpissr:
>
> +  aka: []
>
>
> MPI-SSR
> MPISSR
> MPI SSR
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -872,6 +924,40 @@ natautoumexico:
>
>    state: ''
>
>    street: Av Universidad 3000, Cd. Universitaria, Coyoacán
>
>    zip:  <https://www.google.com/maps/search/Av+Universidad+3000,+Cd.+Universitaria,+Coyoac%C3%A1n%0D%0D%0A+++zip:++04510?entry=gmail&source=g>'04510 <https://www.google.com/maps/search/Av+Universidad+3000,+Cd.+Universitaria,+Coyoac%C3%A1n%0D%0D%0A+++zip:++04510?entry=gmail&source=g>'
>
> +navalresearchlaboratory:
>
> +  aka: []
>
> +  city: washington
>
> +  country: USA
>
> +  day: 17
>
> +  month: 3
>
> +  name: naval research laboratory
>
> +  state: DC
>
> +  updated: 2020-03-17 01:52:10.463834
>
> +  uuid: 87adc340-8b39-4e02-a154-1abc47758dc3
>
> +  year: 2020
>
> +  zip: '20375'
>
> +navigant:
>
> +  aka: []
>
> +  city: new york city
>
>
> New York
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -1107,6 +1316,18 @@ rutherfordappletonlaboratory:
>
>    updated: 2020-02-14 19:55:12.462021
>
>    uuid: d48cb08d-d9a9-4f86-b12d-5751df57006a
>
>    year: 2020
>
> +sandianationallaboratories:
>
> +  aka: []
>
>
> SNL
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -1142,6 +1396,19 @@ srtmu:
>
>    name: Swami Ramanand Teerth Marathwada University
>
>    state: Maharashtra
>
>    zip: ''
>
> +stonybrooku:
>
> +  aka:
>
> +    - sbu
>
> +  city: stony brook
>
> +  country: usa
>
> +  day: 17
>
> +  month: 3
>
> +  name: stony brook university
>
>
> State University of New York, Stony Brook
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -1142,6 +1396,19 @@ srtmu:
>
>    name: Swami Ramanand Teerth Marathwada University
>
>    state: Maharashtra
>
>    zip: ''
>
> +stonybrooku:
>
> +  aka:
>
> +    - sbu
>
>
> add
> suny-sb
> sunysb
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Billingegroup/rg-db-public/pull/221#pullrequestreview-375878437>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOPDY4AYPWWT727O3RRD2K3RH5D5PANCNFSM4LM6JURA>
> .
>

sbillinge commented:
Hi Berrak,

No worries at all!  Let's do the rebase quickly by Zoom.   Are you free now?

S

berrakozer commented:
Yes i’m calling right away!

On Tue, Mar 17, 2020 at 12:28 Simon Billinge <notifications@github.com>
wrote:

> Hi Berrak,
>
> No worries at all! Let's do the rebase quickly by Zoom. Are you free now?
>
> S
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Billingegroup/rg-db-public/pull/221#issuecomment-600167666>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOPDY4BG4YYAWPTP3D6TVKLRH6QJ7ANCNFSM4LM6JURA>
> .
>

sbillinge commented:
I sent a zoom link.

On Tue, Mar 17, 2020 at 12:30 PM berrakozer <notifications@github.com>
wrote:

> Yes i’m calling right away!
>
> On Tue, Mar 17, 2020 at 12:28 Simon Billinge <notifications@github.com>
> wrote:
>
> > Hi Berrak,
> >
> > No worries at all! Let's do the rebase quickly by Zoom. Are you free now?
> >
> > S
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/Billingegroup/rg-db-public/pull/221#issuecomment-600167666
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AOPDY4BG4YYAWPTP3D6TVKLRH6QJ7ANCNFSM4LM6JURA
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Billingegroup/rg-db-public/pull/221#issuecomment-600168665>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAOWUMAHSMGSMIIEW52AYTRH6QQZANCNFSM4LM6JURA>
> .
>


-- 
Professor Simon Billinge
Columbia University

berrakozer commented on db/institutions.yml:
```diff
@@ -1023,6 +1142,52 @@ pekingu:
   zip: '100871'
+pennstateu:
+  aka: []
+  city: Pennsylvania
```
@sbillinge do you want City College to be an aka or it the name itself? Currently the name is fixed as Pennsylvania State University. I will be waiting to hear from your for the necessary City College adjustment, proceeding with the rest in the meantime.

sbillinge commented:
Whoops, I thought that was the name of the city but actually it is "State
College" in Pennsylvania.

On Tue, Mar 17, 2020 at 3:21 PM berrakozer <notifications@github.com> wrote:

> *@berrakozer* commented on this pull request.
> ------------------------------
>
> In db/institutions.yml:
>
> > @@ -1023,6 +1142,52 @@ pekingu:
>    state: ''
>    street: 5 Yiheyuan Road
>    zip: '100871'
> +pennstateu:
> +  aka: []
> +  city: Pennsylvania
>
> @sbillinge <https://github.com/sbillinge> do you want City College to be
> an aka or it the name itself? Currently the name is fixed as Pennsylvania
> State University. I will be waiting to hear from your for the necessary
> City College adjustment, proceeding with the rest in the meantime.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Billingegroup/rg-db-public/pull/221#discussion_r393915008>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAOWUKBG7B3BUKOJKXEQJLRH7EULANCNFSM4LM6JURA>
> .
>


-- 
Professor Simon Billinge
Columbia University

berrakozer commented:
@sbillinge I added State College as an aka for Pennstate University. For the rest I fixed everything you asked for. If there is anything I missed or an additional thing I can do please let me know.

berrakozer commented:
@sbillinge something's wrong. I looked at details but it is not showing me what is wrong.

sbillinge commented:
thanks @berrakozer super!

sbillinge commented:
I will fix the failing test.  Probably SNL is not a list but needs to b.

berrakozer commented:
@sbillinge perfect! please let me know if you need anything else!
